packages/augur-ui/src/modules/portfolio/components/favorites/favorites.tsx

```diff
@@ -90,12 +90,17 @@ export default class Favorites extends Component<FavoritesProps> {
       customClass = favoriteStyles.WatchlistEmptyDisplay;
     }
 
+    let sortByStyles = {}
+    if (getTheme() === THEMES.TRADING) {
```

Comment:
let's move all this logic out and for the `customClass` we put in the `min-width: 10.625rem` in the class for trading and then remove it for betting/sports

Reply:
Updated. Please let me know if I miss something. Thanks.